_docker/drupal/drupal-filesystem/web/themes/custom/rhdp/templates/assembly/assembly--embedded-content.html.twig

```diff
@@ -2,5 +2,5 @@
   <section{{ attributes.addClass('assembly') }}{{ audience_selection }}>
     <a href="{{ embed_url }}" data-iframely-url></a>
   </section>
-  <script async charset="utf-8" src="//cdn.iframe.ly/embed.js?key=71b6124e790b6d7e8cdcba47b2a69ccb"></script>
```

Comment:
@rogeralloymagnetic Is there any security risk for us having this in the open in a publicly clonable Git repo? Is this key specific to this project and paid for?

cheers,

Rob

Reply:
@robpblake I'm following the iFramely best practices for embed.js as outlined here: 
https://iframely.com/docs/embedjs#add-embed-js-to-your-page

And have setup whitelists for developers.redhat.com & localhost:8888 as outlined here:
https://iframely.com/docs/allow-origins

This is a hashed key specific to this paid account, however it would not be able to run on any other domain. As for the localhost whitelist addition, I could not think of another way to allow us to test it in our local dev environments and assumed that the need to pull the protected docker image would help against this. 

I guess in thinking through it again, it is possible for someone to use the key to make calls from any local environment on port 8888, so maybe it's worth discussing what to whitelist vs what will work in local dev. That being said, we can whitelist a test domain as well but you'd have to push it there to see embeds working properly.